subnetGroups.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
I think at this point we may want to consider splitting out the generated json files from the configuration json files to make it clear which are generated from actions and which are the source configs.  Just as a reviewer, I had to double check if this was something generated or not.

I'm thinking either having all the action outputs in their own folder or a clear naming convention for them.

Reply:
Yes, agreed.  Mind if we do that in a separate PR?  I can get one together for that now, but currently this PR has zero breaking changes for the wallet team, and doing that would introduce breaking changes and cause hold-ups on this task.

Reply:
Yep sounds good!